zebrad/src/components/mempool/storage.rs

```diff
@@ -410,11 +410,23 @@ impl Storage {
         self.verified.transactions().map(|tx| tx.id)
     }
 
-    /// Returns the set of [`UnminedTx`]s in the mempool.
+    /// Returns an iterator over the [`UnminedTx`]s in the mempool.
+    //
+    // TODO: make the transactions() method return VerifiedUnminedTx,
+    //       and remove the full_transactions() method
     pub fn transactions(&self) -> impl Iterator<Item = &UnminedTx> {
         self.verified.transactions()
     }
 
+    /// Returns an iterator over the [`VerifiedUnminedTx`] in the set.
+    ///
+    /// Each [`VerifiedUnminedTx`] contains an [`UnminedTx`],
+    /// and adds extra fields from the transaction verifier result.
+    #[allow(dead_code)]
```

Comment:
```suggestion
    #[cfg(feature = "getblocktemplate-rpcs")]
```

Reply:
We'll need this for tickets #5335 and #5336, which are independent of the  `getblocktemplate-rpcs` feature.